Dockerfile

```diff
@@ -32,5 +30,7 @@ RUN TORCH_CUDA_ARCH_LIST="3.5 5.2 6.0 6.1+PTX" TORCH_NVCC_FLAGS="-Xfatbin -compr
     CMAKE_PREFIX_PATH="$(dirname $(which conda))/../" \
     pip install -v .
 
+RUN git clone https://github.com/pytorch/vision.git && cd vision && pip install -v .
```

Comment:
In devel dockerfile, pytorch is compiled from master, so it makes sense to have the latest torchvision too. Runtime dockerfile installs latest conda packages.